docs/design-docs/arm/arm.md

```diff
@@ -0,0 +1,86 @@
+# Epiphany Platform backup design document
+
+Affected version: 0.10.x/0.11.x
+
+## Goals
+
+This document outlines an aproach to add (partial) ARM support to Epicli. Not finalized requirements so far:
+
+- ARMv8/ARM64 architecture
+- Centos 7 as operating system
+- "any" provider as we do not want to provide ARM infrastructure on any cloud providers jet through Terraform
+- Epiphany components: K8s, Kafka, ELK, Grafana, Prometheus, Kibana, HAProxy, Keycloak, PostgreSQL
+
+## Approach
+
+The 2 high level approaches that have been opted so far:
+
+1. Add “architecture” flag when using Epicli
+2. Add new OS (CentosARM64 fe.)
+
+Have 2 big disadvanges from the start:
+
+1. Will require an additional input which makes things more confusing as they will need supply not only the OS but also Architecture for (offline) install. This should not be needed as we can detect the architecture we are working on, on all required levels.

```

Comment:
Are you assuming that the server where we will be downloading packages for install will have the same architecture as the target one?

Reply:
Yes, the download of requirements is already dependent on OS and for architecture the requirement will be the same. Packages you need to pull are architecture dependent.